style/index.less

```diff
@@ -0,0 +1,18 @@
+.content {
+  margin: 20px 10%;
+
+  [data-wikipedia-preview] {
+    background-color: #eaf3ff;
+    padding: 2px 2px 2px 2px;
+    font-family: 'Linux Libertine','Georgia','Times',serif;
+    border-radius: 4px;
+    border: solid 1px rgba(51, 102, 204, 0.48);
+
+  }
+
+  [data-wikipedia-preview]:after {
+    content: url(/images/info_24_px.svg);
```

Comment:
This URL makes sense in the context of the dev server but this is a standalone library. At runtime, this URL will try to be resolved relative to the partners website.

Reply:
You’re right, we could feed it a relative path then

Reply:
Remember that this library is distributed as a single js file. This icon needs to be embedded in it.

Reply:
I see, I just tested doing this and it's working for me in Firefox, Chrome, Safari. Would this be an appropriate solution though? Haven't committed it yet. 

```
content: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='20' height='20' viewBox='0 0 20 20'><g fill='none' fill-rule='evenodd'><path d='M0 0L20 0 20 20 0 20z'/><path fill='%2336C' fill-opacity='.7' fill-rule='nonzero' d='M10.556 1.111c-4.6 0-8.334 3.733-8.334 8.333s3.734 8.334 8.334 8.334 8.333-3.734 8.333-8.334-3.733-8.333-8.333-8.333zm.833 12.5H9.722v-5h1.667v5zm0-6.667H9.722V5.278h1.667v1.666z'/></g></svg>");
```

https://stackoverflow.com/questions/10768451/inline-svg-in-css

Reply:
I think so but webpack or less should be able to do it for us.

https://github.com/atlassian/less-plugin-inline-svg

Reply:
Agree using a plugin would be a cleaner approach. I'm following [this documentation](https://webpack.js.org/loaders/less-loader/#plugins) to enable it but I'm not getting any results or errors. It looks like [there have been issues](https://github.com/webpack-contrib/less-loader/issues/188) with `less-loader` and plugins before that might be affecting here, unless I'm missing something. Any ideas? I can file an issue with `less-loader` or `less-plugin-inline-svg` folks to try our luck there.

Reply:
Yeah, let it go if it doesn't work. We'll figure it out later if we have more image and it becomes an annoyance to manage them inline.